beacon-chain/sync/error.go

```diff
@@ -56,6 +55,9 @@ func writeErrorResponseToStream(responseCode byte, reason string, stream libp2pc
 		log.WithError(err).Debug("Could not generate a response error")
 	} else if _, err := stream.Write(resp); err != nil {
 		log.WithError(err).Debugf("Could not write to stream")
+	} else {
+		// If sending the error message succeeded, close to send an EOF.
+		stream.Close()
```

Comment:
Please handle this error from stream.Close(). (Static analysis will not allow the build to complete with unhandled errors)